### PR TITLE
Update the panasonic_viera component documentation

### PR DESCRIPTION
**Pull request by joogps**
## Proposed change

Updating the documentation for the panasonic_viera component.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/33829
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_integrations/panasonic_viera.markdown [hunk 1]
```diff
@@ -11,79 +11,52 @@ ha_domain: panasonic_viera
...
 
 ```yaml
 # Example configuration.yaml entry
-media_player:
-  - platform: panasonic_viera
-    host: 192.168.0.10
+panasonic_viera:
+  host: 192.168.1.10
+  name: Living Room TV
+  turn_on_action:
```
Comment (frenck) [inline source/_integrations/panasonic_viera.markdown]: We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.

File: source/_integrations/panasonic_viera.markdown [hunk 2]
```diff
@@ -11,79 +11,52 @@ ha_domain: panasonic_viera
 
 The `panasonic_viera` platform allows you to control a Panasonic Viera TV.
 
-Currently known supported models:
+### Setup
```
Comment (frenck) [inline source/_integrations/panasonic_viera.markdown]:
```suggestion
## Configuration

```

File: source/_integrations/panasonic_viera.markdown [hunk 3]
```diff
@@ -11,79 +11,52 @@ ha_domain: panasonic_viera
...
-- TX55ASW654
-
-If your model is not on the list then give it a test, if everything works correctly then add it to the list on [GitHub](https://github.com/home-assistant/home-assistant.io/blob/current/source/_integrations/panasonic_viera.markdown).
-
-Some Panasonic Viera TVs allow Home Assistant to turn them on, if you specify the MAC address with `mac:`.
-
-Note that your TV has to reside in the same network as your Home Assistant instance for this platform to work. If you have multiple network interfaces on your Home Assistant instance, you may need to specify the `broadcast_address`.
-
-To add a TV to your installation, add the following to your `configuration.yaml` file:
+### YAML configuration
```
Comment (frenck) [inline source/_integrations/panasonic_viera.markdown]:
```suggestion
## Manual configuration

```

File: source/_integrations/panasonic_viera.markdown [hunk 4]
```diff
@@ -11,79 +11,52 @@ ha_domain: panasonic_viera
...
-  type: boolean
-name:
-  description: The name you would like to give to the Panasonic Viera TV.
-  required: false
-  default: Panasonic Viera TV
-  type: string
+  type: list
 {% endconfiguration %}
 
+When you restart Home Assistant, make sure the TV is turned on and connected to your local network. If your TV needs to be paired, you'll need to go to **Configuration > Integrations** to type the PIN code that will be displayed on it and finish setup.
```
Comment (frenck) [inline source/_integrations/panasonic_viera.markdown]:
```suggestion
When you restart Home Assistant, make sure the TV is turned on and connected to your local network. If your TV needs to be paired, you'll need to go to **Configuration > Integrations** to type the PIN code that will be displayed on it and finish the setup.
```

**Review by frenck — CHANGES_REQUESTED** [review]
Thanks for the PR @joogps!

Got a couple of small comments/suggestions. Could you take a look? Thanks! 👍

**Comment by joogps** [comment]
Great, of course I will! Just give me a sec...

**Comment by joogps** [comment]
Done! @frenck

File: source/_integrations/panasonic_viera.markdown [hunk 5]
```diff
@@ -11,79 +11,60 @@ ha_domain: panasonic_viera
...
+## Manual configuration
+If you prefer to use YAML to set up your Panasonic Viera TV, you can still do it. It also allows for some extra settings.
 
 ```yaml
 # Example configuration.yaml entry
-media_player:
-  - platform: panasonic_viera
-    host: 192.168.0.10
+panasonic_viera:
+  host: 192.168.1.10
```
Comment (frenck) [inline source/_integrations/panasonic_viera.markdown]: Missed one 😓

```suggestion
  host: YOUR_TV_IP
```

We use placeholders for these. Sorry about that.

Comment (joogps) [reply source/_integrations/panasonic_viera.markdown]: it's ok lol

should I add a description to explain what YOUR_TV_IP should be or that's self-explanatory enough?

Comment (klaasnicolaas) [reply source/_integrations/panasonic_viera.markdown]: Precisely because it is a placeholder, it should already be clear that you have to adjust this 😉

**Review by klaasnicolaas — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged 👍

**Comment by joogps** [comment]
Parent merged! home-assistant/core#33829

**Review by klaasnicolaas — DISMISSED** [review]

**Review by frenck — APPROVED** [review]

**Pull request merged**